src/ray/core_worker/object_interface.cc

```diff
@@ -3,18 +3,48 @@
 #include "ray/common/ray_config.h"
 #include "ray/core_worker/object_interface.h"
 #include "ray/core_worker/store_provider/local_plasma_provider.h"
+#include "ray/core_worker/store_provider/memory_store_provider.h"
 #include "ray/core_worker/store_provider/plasma_store_provider.h"
 
 namespace ray {
 
+void ObjectIdsByStoreProvider(
```

Comment:
`GroupObjectIdsByStoreProvider`? and document

Reply:
sure.